src/main/java/org/takes/rs/RsWithBody.java

```diff
@@ -143,4 +176,19 @@ public InputStream body() {
         );
     }
 
+    /**
+     * Forms a list with header attributes from response.
+     * @param head Response head
+     * @return List of header attributes
+     */
+    private static List<String> getHeadAttributes(final Iterable<String> head) {
+        final Iterator<String> itr = head.iterator();
+        // @checkstyle ConditionalRegexpMultilineCheck (1 line)
+        final List<String> headerAttrs = new ArrayList<String>();
```

Comment:
If you don't know the initial size of array and don't care about search by index consider using ```LinkedList``` instead of ```ArrayList```.  Thus, you will get rid of checkstyle suppression.